omnigibson/sensors/vision_sensor.py

```diff
@@ -130,11 +130,9 @@ def __init__(
             load_config=load_config,
         )
 
-    def _load(self, simulator=None):
+    def _load(self):
         # Define a new camera prim at the current stage
-        stage = get_current_stage()
-        prim = UsdGeom.Camera.Define(stage, self._prim_path).GetPrim()
-        return prim
+        return UsdGeom.Camera.Define(get_current_stage(), self._prim_path).GetPrim()
```

Comment:
can we add a comment here as to why we don't directly do `og.sim.stage` like the other cases?

Reply:
done